service/controller/v2/resources/vaultcrt/current.go

```diff
@@ -67,6 +73,16 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 	return secret, nil
 }
 
+func (r *Resource) checkCertType(customObject v1alpha1.CertConfig) bool {
```

Comment:
I'm having a hard time following this. So `certs.AllCerts` are the certs we support now which excludes Calico?

I'd like to see some godoc here on what we are checking and why we are checking it.

Reply:
@rossf7 I added godoc on private function and just above the function usage too, hope that helps.